R/Rules-class.R

```diff
@@ -925,6 +925,65 @@ StoppingMTDdistribution <- function(target,
                              prob=prob)
 }
 
+#' Stop based on precision of MTD calculated as CV(MTD)
+#' 
+#' @description `r lifecycle::badge("experimental")`
+#' 
+#'
+#' @slot target (`numeric`)\cr toxicity target of MTD
+#' @slot threshCV (`numeric`)\cr threshold for CV to be considered accurate enough
+#'  to stop the trial
+#' 
+#' @example examples/Rules-class-StoppingMTDCV.R
+#' @export
+.StoppingMTDCV <-
+  setClass(
+    Class = "StoppingMTDCV",
+    representation(
+      target = "numeric",
+      threshCV = "numeric"
+    ),
+    prototype(
+      target = 0.33,
+      threshCV = 40
+    ),
+    contains = "Stopping",
+    validity =
+      function(object) {
+        o <- Validate()
+        o$check(
+          is.probability(object@target,
+            bounds = FALSE
+          ),
+          "target must be probability > 0 and < 1"
+        )
+        o$check(
+          is.probability(object@threshCV/100,
+                         bounds = FALSE
+          ),
```

Comment:
```suggestion
          is.probability(object@threshCV/100, bounds = FALSE),
```

Reply:
Changed. I just used what 'Style selection' have returned without further editing.

Reply:
thx - yeah that gives a good start, then some manual polishing is sometimes needed